snorkel/learning/utils.py

```diff
@@ -74,6 +74,51 @@ def score(self, test_marginals, train_marginals=None, b=0.5, set_unlabeled_as_ne
 
 class MentionScorer(Scorer):
     """Scorer for mention level assessment"""
+
+    def score_b(self, test_marginals, b, get_candidates=False, set_unlabeled_as_neg=True, set_at_thresh_as_neg=True):
```

Comment:
Modularity. Reuse of code. Etc. This is called both by `pr_curve` (over multiple threshold values) and by `score`. Can rename to `_score_threshold`